src/steamship/data/file.py

```diff
@@ -254,6 +254,9 @@ def list(client: Client) -> ListFileResponse:
             expect=ListFileResponse,
         )
 
+    def append_block(self, text: str = None, tags: List[Tag] = None) -> Block:
+        return Block.create(self.client, self.id, text, tags)
```

Comment:
```suggestion
        return Block.create(self.client, file_id=self.id, text=text, tags=tags)
```

Would use kwargs here since I didn't understand why we used self.id here.